router/voteCallbacks.go

```diff
@@ -257,13 +305,136 @@ func (r *Router) getEnvelopeList(request routerRequest) {
 	if request.ListSize == 0 {
 		request.ListSize = MaxListSize
 	}
-	nullifiers := r.vocapp.State.EnvelopeList(request.ProcessID, request.From, request.ListSize, true)
+	envelopes, err := r.Scrutinizer.GetEnvelopes(request.ProcessID)
```

Comment:
This is extremly costly and does not scale, no way we can return such list. We should instead return the nullifiers (as it was before) and get the envellope content only when requested individually.